source/math/FloatVectorList.ooc

```diff
@@ -381,4 +381,80 @@ FloatVectorList: class extends VectorList<Float> {
 			result add(this[i] clamp(floor, ceiling))
 		result
 	}
+	findMaxIndex: func -> (Int, Float) {
```

Comment:
There is property "maxValue" that returns value, but it doesn't index. I was wondering if it's possible to return both. What do you think?